51_fl_diff_shipment.py

```diff
@@ -0,0 +1,258 @@
+    * 100000
+)
+florida_prescriptions
+# calculating and displaying total number of drug prescriptions that took place in Florida. results are grouped and displayed year and county wise
+
+florida_prescriptions_result = (
+    florida_prescriptions.groupby(["Year", "county_test"])["shipment_per_100k"]
+    .sum()
+    .reset_index()
+)
+# florida_prescriptions_result = florida_prescriptions.groupby(["Year"])["shipment_per_100k"].sum().reset_index()
+
+florida_prescriptions_result
+
+# creating a new dataset, that contains drug related deaths in other neighboring states of Florida - Alabama, Georgia, Mississippi, South Carolina and Tennessee
+
+other_states_shipment = prescriptions_reduced_copy[
+    (prescriptions_reduced_copy["StateName"] != "Florida")
+]
+# other_states_shipment = prescriptions_reduced_copy[(prescriptions_reduced_copy["StateName"] == "Alabama")|(prescriptions_reduced_copy["StateName"] == "Georgia")|(prescriptions_reduced_copy["StateName"] == "Mississippi")|(prescriptions_reduced_copy["StateName"] == "South Carolina")|(prescriptions_reduced_copy["StateName"] == "Tennessee")]
+
+other_states_shipment
+
+other_states_shipment_copy = other_states_shipment.copy()
+
+other_states_shipment_copy["shipment_per_100k"] = (
+    other_states_shipment_copy["QUANTITY"] / other_states_shipment_copy["Population"]
+) * 100_000
+other_states_shipment_copy["shipment_per_100k"] = (
+    (
+        other_states_shipment_copy["dos_str"]
+        * other_states_shipment_copy["DOSAGE_UNIT"]
+        * other_states_shipment_copy["MME_Conversion_Factor"]
+    )
+    / (other_states_shipment_copy["Population"])
+    * 100000
+)
+other_states_shipment_copy
+# calculating and displaying results for drug related deaths - year, state and county wise
+
+other_states_result = (
+    other_states_shipment_copy.groupby(["Year", "StateName", "county_test"])[
+        "shipment_per_100k"
+    ]
+    .sum()
+    .reset_index()
+)
+other_states_result.head()
+# calculating yearwise drug related deaths results for Alabama
+alabama_results = (
+    other_states_shipment_copy[other_states_shipment_copy["StateName"] == "Alabama"]
+    .groupby("Year")["shipment_per_100k"]
+    .mean()
+)
+alabama_results
+# calculating yearwise drug related deaths results for Georgia
+georgia_results = (
+    other_states_shipment_copy[other_states_shipment_copy["StateName"] == "Georgia"]
+    .groupby("Year")["shipment_per_100k"]
+    .mean()
+)
+georgia_results
+# calculating yearwise drug related deaths results for Mississippi
+mississippi_results = (
+    other_states_shipment_copy[other_states_shipment_copy["StateName"] == "Mississippi"]
+    .groupby("Year")["shipment_per_100k"]
+    .mean()
+)
+mississippi_results
+# calculating yearwise drug related deaths results for South Carolina
+south_carolina_results = (
+    other_states_shipment_copy[
+        other_states_shipment_copy["StateName"] == "South Carolina"
+    ]
+    .groupby("Year")["shipment_per_100k"]
+    .mean()
+)
```

Comment:
This code will need to be modified since you need to use the new comparison states for Florida